### ISSUE TITLE
Update version in addon.xml

### ISSUE THREAD

**Issue by antonsoroko** [fugkco/script.elementum.jackett#31]
Hi.
Looks like you still use version "0.0.1" while in reality you have version 3.0 already:
https://github.com/fugkco/script.elementum.jackett/blob/fceaee5ebe724da8b6da6e7840014ef70c587081/addon.xml#L3

I suggest to update `addon.xml` very time you make a release.

**Comment by fugkco** [comment]
[The version gets updated during build-time.](https://github.com/fugkco/script.elementum.jackett/blob/master/scripts/update-version.py)

**Comment by antonsoroko** [comment]
then i guess something else is wrong, since it stuck on version 0.0.1 in `repository.elementumorg`.

**Comment by fugkco** [comment]
Interesting! Have you validated that the actual files are using the latest version maybe? If not, it's probably an issue with the repository, so you have done the right thing to open an issue there.

**Comment by fugkco** [comment]
I'll leave it open for now until more is known.

**Comment by antonsoroko** [comment]
i checked content of poetry.lock and it is old, thus i opened a new issue.

i guess people either use old version or download new versions manually.

**Comment by vasilky3** [comment]
you can see last release archive have version.
`<addon id="script.elementum.jackett" name="Elementum [COLOR FF0291F7]Jackett[/COLOR]" version="3.0.0" provider-name="fugkco">`
I don't know how does it uploads to repo. if it is?

**Comment by antonsoroko** [comment]
@vasilky3 See the issue I linked here. Elementum uses dynamic repo, it uses GitHub API to identify latest version of addon. It works fine for all "official" elementum addons, but for some reason does not work for this addon.

**Comment by elgatito** [comment]
Yes, that is because repository expects a new version in addon.xml.

That is the way how Kodi identifies a newer version. It does not look on zip file name, but reads addon.xml to parse version.

@fugkco Would be great if you could update a version with addon.xml changes to list actual version name.

Or we can update repository code to get the latest tag to identify version, but that is another story..

**Comment by vasilky3** [comment]
@elgatito, sorry been watching from phone. forgot to press "show all"

**Comment by vasilky3** [comment]
Moreover in las PR i changed naming for *.zip in release. to {name}-{ver}
https://github.com/fugkco/script.elementum.jackett/pull/32/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R13
So we will need to fix it in 
https://github.com/ElementumOrg/repository.elementumorg/blob/5506e2731eb24c7f9b1082ab491bf0e2d85593a0/resources/repository.json#L24-L27

**Comment by elgatito** [comment]
@vasilky3 

> Moreover in las PR i changed naming for *.zip in release. to {name}-{ver}

Yes, since repository is not reading older releases, only the last/current one, so we will need to change pattern for release url.

**Comment by fugkco** [comment]
@elgatito I'm not sure if I'm following. If the version is updated in addon.xml, why does the URL need to change? The GitHub release URL already contains the version number, so I prefer to keep it out of the filename, as otherwise it renders the latest endpoint of GitHub's releases API useless.

Anyway, I'll make some updates soon so it commits the correct version to the repo as part of a release!

**Comment by antonsoroko** [comment]
@fugkco it was answer to comment above. In that PR the name of zip file will be changed for your git repo, thus the name of zip file should be changed in kodi's repo config as well (in URL).

**Comment by elgatito** [comment]
> @elgatito I'm not sure if I'm following. If the version is updated in addon.xml, why does the URL need to change? The GitHub release URL already contains the version number, so I prefer to keep it out of the filename, as otherwise it renders the latest endpoint of GitHub's releases API useless.

@fugkco Repository addon is not using Github API to get correct url to release assets.
We only get `addon.xml` file from current state of repository, read version from it, and then construct URL to artifacts dynamically, so if your format of release files changes - repository needs to change contruction format for your repository.
